### ISSUE TITLE
9.2 - php warning viewing created tickets tab of a user

### ISSUE THREAD

**Issue by btry** [glpi-project/glpi#2527]
Attempt to read the field is_template in a User object.

The user class does not handles templates.

```
2017-07-28 10:26:07 [2@workstation]
  *** PHP Notice(8): Undefined index: is_template
  Backtrace :
  inc/ticket.class.php:5610                          
  inc/ticket.class.php:727                           Ticket::showListForItem()
  inc/commonglpi.class.php:482                       Ticket::displayTabContentForItem()
  inc/commonglpi.class.php:449                       CommonGLPI::displayStandardTab()
  ajax/common.tabs.php:96                            CommonGLPI::displayStandardTab()
```

**Comment by btry** [comment]
Seems related to the fix for #2322 in commit 4f8639942c1390624730768e0abb7e68cad4b43d

**Comment by trasher** [comment]
Fixed, see 3d960247f27b80c146809b8c18dee0380e1076ef
